packages/doenetml/rust/src/Viewer/PageViewer.jsx

```diff
@@ -718,6 +718,8 @@ export default function PageViewer(props) {
     function startCore() {
         console.log(`send message to create core ${pageNumber}`);
 
+        // TODO - for compatiblity with firefox we probably want to make this "classic" instead
```

Comment:
Yes, this should also be changed. I also don't understand why it isn't loading `doenetGlobalConfig.doenetWorkerUrl`.

Reply:
reverted